MaxiOps/00partnerLM/TokenLogic/ARB/GHO_2024-07-26.json

```diff
@@ -0,0 +1,46 @@
+{
+  "version": "1.0",
+  "chainId": "42161",
+  "createdAt": 1691667397141,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.16.1",
+    "createdFromSafeAddress": "0x9ff471F9f98F42E5151C7855fD1b5aa906b1AF7e",
+    "createdFromOwnerAddress": "",
+    "checksum": ""
+  },
+  "transactions": [
+    {
+      "to": "0xE23eb92f0C76bF47f77F80D144e30F31b98450A9",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "name": "gaugeAddresses",
+            "type": "address[]",
+            "internalType": "address[]"
+          },
+          {
+            "name": "amountsPerPeriod",
+            "type": "uint256[]",
+            "internalType": "uint256[]"
+          },
+          {
+            "name": "maxPeriods",
+            "type": "uint8[]",
+            "internalType": "uint8[]"
+          }
+        ],
+        "name": "setRecipientList",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "gaugeAddresses": "[0x6e654787251ae77fe1873f93a5f220e00afa90e8,0x7d907d8d66b6cad285d4349f507ecdfd11fd23d2]",
+        "amountsPerPeriod": "[18450000000000000000000,5740500000000000000000]",
```

Comment:
- target injector matches
- Does match program on [sheet](https://docs.google.com/spreadsheets/d/1gZOBQ5AadhsU3nMoylEHV73wWsRyOtANbcR_8UaSH98/edit?gid=0#gid=0)

gho/USDC/USDT should be 11481 so it is 5740.5 per week but report only shows first week.

lgtm